src/Raven.Server/Commercial/LicenseManager.cs

```diff
@@ -358,7 +358,8 @@ public async Task ChangeLicenseLimits(string nodeTag, int newAssignedCores, stri
                         InstalledMemoryInGb = installedMemoryInGb,
                         UsableMemoryInGb = usableMemoryInGb,
                         BuildInfo = buildInfo,
-                        OsInfo = osInfo
+                        OsInfo = osInfo,
+                        Modified = newAssignedCores != _licenseStatus.MaxCores
```

Comment:
Not sure that I follow this. If I have a license with 64 cores and a node with 32 cores then it will be always true? How this is 'set by user explicitly'?